packages/aws-cdk-lib/aws-lambda-event-sources/lib/stream.ts

```diff
@@ -73,7 +75,8 @@ export interface StreamEventSourceProps extends BaseStreamEventSourceProps {
    * * Minimum value of 0
    * * Maximum value of 10000
    *
-   * @default - retry until the record expires
+   * @default -1 
+   * retry until the record expires
```

Comment:
This seems like an oversight.

Reply:
Thanks for the review @jonife , This one is -1 as well, [see doc](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-lambda-eventsourcemapping.html#cfn-lambda-eventsourcemapping-maximumrecordageinseconds), adding default here the same reason for maxRecordAge